rskj-core/src/test/java/co/rsk/jsonrpc/JsonRpcErrorResponseTest.java

```diff
@@ -30,10 +30,10 @@
 
     @Test
     public void serializeResponseWithError() throws IOException {
-        String message = "{\"jsonrpc\":\"2.0\",\"id\":48,\"error\":{\"code\":-32603,\"message\":\"Internal error.\"}}";
+        String message = "{\"jsonrpc\":\"2.0\",\"id\":\"48\",\"error\":{\"code\":-32603,\"message\":\"Internal error.\"}}";
```

Comment:
I would leave this test as it was, since it was testing with the ID as an integer (should return an integer) and then would add another test below it but setting the ID as string (same thing as the modified version, basically).

Reply:
This coherence of type is now been assured by the RskWebSocketJsonRpcHandler.getJsonWithTypedId